packages/graphql-transformer-core/src/collectDirectives.ts

```diff
@@ -66,14 +65,7 @@ export function collectDirectivesByType(sdl: string): Object {
     return {};
   }
   let doc;
-  try {
-    doc = parse(sdl);
-  } catch (e) {
```

Comment:
This catch block is unreachable in the current state.

Reply:
Is this because of something specific in the `parse` impl? It's not obvious why this would be unreachable, unless I'm missing something obvious. Also if you're no longer adding the try block, etc. can the let and parse be consolidated into a single line (or better yet, can it be made a const?)

Reply:
Sorry, I should have added some for details. It is unreachable in the scenario of running from the CLI. The code would fail somewhere else before reaching this `parse` to throw on.

But now I am doubting myself, so I'll take another look.